csrc/quantization/cutlass_w8a8/grouped_mm_c3x.cuh

```diff
@@ -0,0 +1,188 @@
+#pragma once
+
+#include "cutlass/cutlass.h"
+
+#include "cutlass/gemm/collective/collective_builder.hpp"
+#include "cutlass/epilogue/collective/collective_builder.hpp"
+#include "cutlass/gemm/device/gemm_universal_adapter.h"
+
+#include "cutlass_extensions/epilogue/scaled_mm_epilogues_c3x.hpp"
+#include "cutlass_extensions/common.hpp"
+
+using namespace cute;
+
+#if defined __CUDA_ARCH__ && __CUDA_ARCH__ >= 900
+  #define ENABLE_SM90_KERNEL_LEVEL 1
+#endif
+
```

Comment:
We will need a function like `bool cutlass_group_gemm_supported(int64_t cuda_device_capability)` that checks the `CUDA_VERSION` to report if `cutlass_group_gemm` is supported.